assembl/static2/js/app/utils/utilityManager.jsx

```diff
@@ -72,6 +74,21 @@ export const closeModal = () => {
   modalManager.component.setState({ showModal: false });
 };
 
+export const openShareModal = (title, routerParams, elementId, social, isFooter, footer) => {
```

Comment:
As these entities are rather long and in each case not needed, pass an `option` object around, and take the appropriate keys in the method below. Would be easier :)

Reply:
I agree